src/components/AppForm.js

```diff
@@ -0,0 +1,92 @@
+import { LitElement, html, css } from 'lit-element';
+
+import './AppButton';
+
+class AppForm extends LitElement {
+
+  static get styles() {
+    return css`
+      .wrapper {
+        padding: 20px;
+        border: 1px solid #eee;
+        background-color: #fff;
+        border-radius: 6px;
+      }
+      div {
+        padding: 6px 0;
+      }
+      input[type="text"], textarea {
+        width: 100%;
+        padding: 6px 8px;
+        box-sizing: border-box;
+      }
+      textarea {
+        resize:vertical;
+      }
+    `;
+  }
+
+  constructor() {
+    super();
+
+    this.title = '';
+    this.description = '';
+  }
+
+  /**
+   * Handle change in input value.
+   * @param {Object} e Input event object. 
+   */
+  handleInputChange = (e) => {
+    this[e.target.name] = e.target.value;
+  }
+
+  handleAddTodo = () => {
+
+    if (this.title) {
+      const todo = {
+        id: Date.now(),
+        title: this.title,
+        description: this.description,
+        active: true
+      }
+
+      this.title = '';
+      this.description = '';
```

Comment:
Better to extract these in a method named `resetInput`, so that we won't have to repeat yourself all the time. Keep in mind the [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) principle.